gst/nnstreamer/include/tensor_typedef.h

```diff
@@ -62,6 +62,24 @@
  */
 #define GST_TENSOR_TYPE_ALL "{ float32, float64, int64, uint64, int32, uint32, int16, uint16, int8, uint8 }"
 
+/**
+ * @brief Possible tensor formats
+ */
+#define GST_TENSOR_FORMAT_ALL "{ static, flexible, sparse } "
+
+/**
+ * @brief Caps string for the caps template of tensor stream.
+ * format should be a string that describes the data format, or possible formats of incoming tensor.
+ *
+ * If the data format is static, another tensor information should be described in caps.
+ * - num-tensors: The number of tensors in a GstBuffer.
+ * - types: The list of data type in the tensor. type should be one of types in GST_TENSOR_TYPE_ALL. (types=(string)"unit8,int32,flat32")
+ * - dimensions: The list of dimension in the tensor. dimension should be a formatted string with rank NNS_TENSOR_RANK_LIMIT. (dimensions=(string)"dim1:dim2:dim3:dim4,dim1:dim2:dim3:dim4")
+ */
+#define GST_TENSOR_CAPS_MAKE(format) \
+    NNS_MIMETYPE_TENSOR ", " \
+    "format = (string) " format
```

Comment:
change the macro variable. E.g., format --> fmt

Reply:
updated

Reply:
I modified this again - stringify removed.
gst cannot accept caps template w/quotation mark (e.g.m `other/tensors,format="{ static, flexible, sparse }"` makes error)